website/blog/2021-01/01/_index.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Happy anniversary Gardener! Three years of open source Kubernetes management
+linkTitle: Happy anniversary Gardener! Three years of open source Kubernetes management
+newsSubtitle: January 30, 2021
+type: blog
+publishdate: 2021-01-25
+archivedate: 2021-02-25
+authors:
+- name: Vasu Chandrasekhara
+  email: vasu.chandrasekhara@sap.com
+  avatar: https://avatars.githubusercontent.com/u/11454395?s=400&u=9496275f0718d81bea32068dccb4d9c2d848e592&v=4
+- name: Tim Usner
+  email: tim.usner@sap.com
+  avatar: https://avatars.githubusercontent.com/u/40451181?s=400&u=4df34635cf86b924700ef4152ec8462eeaa35721&v=4
+aliases: ["/blog/2021/01/30/01"]
+---
+
+Happy new year Gardeners!
+As we greet 2021, we also celebrate Gardener’s third anniversary. 
```

Comment:
```suggestion
As we greet 2021, we also celebrate Gardener’s third anniversary. Gardener was born with its first open source commit on 10.1.2018 (its inception within SAP was of course some 9 months earlier :-):
```text
commit d9619d01845db8c7105d27596fdb7563158effe1
Author: Gardener Development Community <gardener.opensource@sap.com>
Date:   Wed Jan 10 13:07:09 2018 +0100

    Initial version of gardener

    This is the initial contribution to the Open Source Gardener project.
...
```
```
If its too edgy, please reformulate :-)